lib/buffer.js

```diff
@@ -65,6 +71,9 @@ function alignPool() {
  * would ever actually be removed.
  **/
 function Buffer(arg, encodingOrOffset, length) {
+  if (!(this instanceof Buffer)) {
+    bufferDeprecationWarning();
+  }
```

Comment:
@jasnell That would emit a warning every time `Buffer()` is called, not only the first time, right?